core/web/resolver/mutation.go

```diff
@@ -828,64 +827,6 @@ func (r *Resolver) UpdateJobProposalSpecDefinition(ctx context.Context, args str
 	return NewUpdateJobProposalSpecDefinitionPayload(spec, err), nil
 }
 
-func (r *Resolver) SetServicesLogLevels(ctx context.Context, args struct {
-	Input struct{ Config LogLevelConfig }
-}) (*SetServicesLogLevelsPayloadResolver, error) {
-	if err := authenticateUser(ctx); err != nil {
-		return nil, err
-	}
-
-	if args.Input.Config.HeadTracker != nil {
-		inputErrs, err := r.setServiceLogLevel(ctx, logger.HeadTracker, *args.Input.Config.HeadTracker)
-		if inputErrs != nil {
-			return NewSetServicesLogLevelsPayload(nil, inputErrs), nil
-		}
-		if err != nil {
-			return nil, err
-		}
-	}
-
-	if args.Input.Config.FluxMonitor != nil {
-		inputErrs, err := r.setServiceLogLevel(ctx, logger.FluxMonitor, *args.Input.Config.FluxMonitor)
-		if inputErrs != nil {
-			return NewSetServicesLogLevelsPayload(nil, inputErrs), nil
-		}
-		if err != nil {
-			return nil, err
-		}
-	}
-
-	if args.Input.Config.Keeper != nil {
-		inputErrs, err := r.setServiceLogLevel(ctx, logger.Keeper, *args.Input.Config.Keeper)
-		if inputErrs != nil {
-			return NewSetServicesLogLevelsPayload(nil, inputErrs), nil
-		}
-		if err != nil {
-			return nil, err
-		}
-	}
-
-	return NewSetServicesLogLevelsPayload(&args.Input.Config, nil), nil
```

Comment:
We can also remove the definition of this constructor and the struct related to it, since it won't be used anymore.

Reply:
File related is here: https://github.com/smartcontractkit/chainlink/blob/e4cc231949af3fc5290cea5ebd6c101759791676/core/web/resolver/log.go#L81